tests/dummy/app/pods/application/template.hbs

```diff
@@ -15,8 +15,11 @@
       <div class='frost-modal-demo-selector-title'>
         Cookbook
         {{#link-to 'simple'}}Basic{{/link-to}}
+        {{#link-to 'simple-loading'}}Basic - Loading{{/link-to}}
```

Comment:
You can put the examples directly into the existing routes - no need to do a duplicate explicitly to demonstrate the loading